lib/exercism/team_membership_invite.rb

```diff
@@ -9,12 +9,18 @@ class TeamMembershipInvite < ActiveRecord::Base
 
   def accept!
     ActiveRecord::Base.transaction do
-      TeamMembership.create!(
-        team_id: team_id,
-        user_id: user_id,
-        inviter_id: inviter_id,
-        confirmed: true
-      )
+      membership = TeamMembership.find_by(team_id: team_id, user_id: user_id)
```

Comment:
Can we use `find_or_initialize_by` here and fill the `inviter_id` and `confirmed` fields right after? This way we can avoid the conditional. :)

We can also use `save!`, just in case something goes wrong.